models/demos/t3000/mixtral8x7b/tests/test_mixtral_rms_norm.py

```diff
@@ -27,25 +27,27 @@
 )
 
 
-@skip_for_wormhole_b0("See GH Issue #10322")
-def test_mistral_rms_norm_inference(t3k_device_mesh, use_program_cache, reset_seeds):
+# @skip_for_wormhole_b0("See GH Issue #10322")
+def test_mixtral_rms_norm_inference(t3k_device_mesh, use_program_cache, reset_seeds):
```

Comment:
[Not a blocker, can be addressed after merge]

maybe all the rmsnorm tests should be in one location (models/common/tests/test_rmsnorm.py), so it is easier to see the coverage for each module.

Reply:
Interesting idea; I'm undecided as to whether we will even want these tests in addition to the existing models/common/testes/test_rmsnorm.py or not. How much value does loading and running all those different weights really add? Either way I definitely want to do a test refactoring pass after this as there's for sure a lot we can improve.